HW1/Task1.1/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿Console.WriteLine("Enter 2 integers: ");
+
+Console.WriteLine("Enter first integer:");
+int a = int.Parse(Console.ReadLine()!);
+
+Console.WriteLine("Enter second integer:");
+int b = int.Parse(Console.ReadLine()!);
+
+Console.Write($"The range of integers from {a} to {b}, which in their duodecimal representation contain exactly two symbols \"A\": ");
+
+
+int[] array = ArrayOfNumbersFromAtoB(a, b);
+
+foreach (int integer in array)
+{
+    if (integer != 0)
+    {
+        Console.Write($"{integer} ");
+    }
+}
+
```

Comment:
I think no need to create an array jut to store values, loop from a to b or from b to a will be enough (save memory)